sdk/keyvault/keyvault-secrets/tests-resources.json

```diff
@@ -51,16 +51,7 @@
                 "verify",
                 "wrapKey"
               ],
-              "secrets": [
-                "backup",
-                "delete",
-                "get",
-                "list",
-                "purge",
-                "recover",
-                "restore",
-                "set"
-              ],
+              "secrets": ["backup", "delete", "get", "list", "purge", "recover", "restore", "set"],
```

Comment:
Nit: I'd leave these on multiple lines for easier diff'ing/merging in the future.

Reply:
I'll undo this change